__tests__/unit/lib/utils/repoGitDiff.test.js

```diff
@@ -177,16 +177,16 @@ describe(`test if repoGitDiff`, () => {
     expect(work).toStrictEqual(expected)
   })
 
-  test('can filter deletion if only ignored is specified files', async () => {
+  test('cannot filter deletion if only ignored is specified files', async () => {
```

Comment:
**remarks:** I included this in the PR as a fix

I think the previous implementation where wrong in its spec, so I changed the test to be closer to what we need
And I think it should be like it is now.
I was not able to remember why it was like this and I assume it was a little mistake that slept through

I tested it functionally and it seems to work ok, it can be tested more

We can discuss it